utils/jwt.go

```diff
@@ -0,0 +1,71 @@
+package utils
+
+import (
+	"errors"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/Real-Dev-Squad/wisee-backend/models"
+	"github.com/golang-jwt/jwt/v5"
+)
+
+/*
+ * GenerateToken generates a JWT token for the user
+ */
+func GenerateToken(user *models.User) (string, error) {
```

Comment:
The tokenExpiryTime variable is formatted using the time.RFC3339Nano format. However, this format includes nanoseconds, which are not supported by all JWT implementations.

A better approach would be to format the tokenExpiryTime variable using the time.RFC3339 format. This format does not include nanoseconds, and it is supported by all JWT implementations.